Sources/TransferMethod/ListTransferMethodTableViewCell.swift

```diff
@@ -41,43 +37,65 @@ final class ListTransferMethodTableViewCell: UITableViewCell {
 
     override func setSelected(_ selected: Bool, animated: Bool) {
         super.setSelected(selected, animated: animated)
-        iconView.backgroundColor = Theme.Icon.backgroundColor
+        imageView?.backgroundColor = Theme.Icon.backgroundColor
     }
 
     override func setHighlighted(_ highlighted: Bool, animated: Bool) {
         super.setHighlighted(highlighted, animated: animated)
-        iconView.backgroundColor = Theme.Icon.backgroundColor
+        imageView?.backgroundColor = Theme.Icon.backgroundColor
     }
 
     // MARK: Theme manager's proxy properties
     @objc dynamic var titleLabelFont: UIFont! {
-        get { return transferMethodTitle.font }
-        set { transferMethodTitle.font = newValue }
+        get { return textLabel?.font }
+        set { textLabel?.font = newValue }
     }
 
     @objc dynamic var titleLabelColor: UIColor! {
-        get { return transferMethodTitle.textColor }
-        set { transferMethodTitle.textColor = newValue }
+        get { return textLabel?.textColor }
+        set { textLabel?.textColor = newValue }
     }
 
     @objc dynamic var subTitleLabelFont: UIFont! {
-        get { return transferMethodSubTitleOne.font }
-        set { transferMethodSubTitleOne.font = newValue
-            transferMethodSubTitleTwo.font = newValue }
+        get { return detailTextLabel?.font }
+        set { detailTextLabel?.font = newValue
+            detailTextLabel?.font = newValue }
     }
 
     @objc dynamic var subTitleLabelColor: UIColor! {
-        get { return transferMethodSubTitleOne.textColor }
-        set { transferMethodSubTitleOne.textColor = newValue
-            transferMethodSubTitleTwo.textColor = newValue}
+        get { return detailTextLabel?.textColor }
+        set { detailTextLabel?.textColor = newValue }
     }
 }
 
 extension ListTransferMethodTableViewCell {
     func configure(configuration: ListTransferMethodCellConfiguration) {
-        transferMethodTitle.text = configuration.transferMethodType
-        transferMethodSubTitleOne.text = configuration.transferMethodCountry
-        transferMethodSubTitleTwo.text = configuration.transferMethodExpiryDate
-        iconView.draw(fontName: configuration.transferMethodIconFont)
+        textLabel?.text = configuration.transferMethodType
+        detailTextLabel?.attributedText = formatSubtitle(transferMethodCountry: configuration.transferMethodCountry,
+                                                         lastFourDigitAccountNumber: configuration
+                                                            .lastFourDigitAccountNumber)
```

Comment:
We don't need to use `attributedText` in this case because of the font and color are simple. 
Suggestion: 
- Remove this function `formatSubtitle` 
- just concatenate the country and the last four digit account number

``` swift 
        var detailText = String(format: "%@\n", configuration.transferMethodCountry)

        if let lastFourDigitAccountNumber = configuration.lastFourDigitAccountNumber {
            detailText = String(format: "%@%@", detailText, lastFourDigitAccountNumber)
        }

        detailTextLabel?.text = detailText
```

Reply:
We are going to create a ticket to have a common ```configuration``` class to deal with both list cell and select cell. This method will be changed as well. Not in this pr